tests/unit/test_requests_handlers.py

```diff
@@ -140,7 +140,8 @@ async def test_post_request_access_code_get_ai_postcode_connection_error_hh_en(
             self.assertLogEvent(cm,
                                 'client failed to connect',
                                 url=self.addressindexsvc_url +
-                                self.postcode_valid)
+                                self.postcode_valid +
+                                self.address_index_epoch_param)
```

Comment:
I would like to see at least one test where the epoch name is actually set. I believe the tests all have empty epoch names.

Reply:
Have now added 3 tests that include setting the epoch